CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contribution Guidelines for iNaturalistReactNative
+Welcome! We're excited that you're interested in contributing to the iNaturalist React Native mobile application. Below you'll find some guidelines that will help you contribute to our project.
+
+We welcome volunteer contributions! This app is still early in its development and a lot of things are in flux, but there's usually something to work on. Please keep the following in mind:
+
+1. Work on [existing issues](https://github.com/inaturalist/iNaturalistReactNative/issues) (though if you have ideas for improvement that aren't directly related to features, let us know). If you'd like to work on something, please leave a comment on that issue and we'll assign you to avoid overlapping work.
+2. Name your branch starting with the issue number and then something descriptive, e.g. `123-login-with-locale-crash`
+3. We try to review pull requests as soon as we can, but that might be up to a week or two
+
+## Getting Started
+1. Fork the repository.
+2. Clone the forked repository to your local machine.
+3. Follow [the README](README.md) to setup your local repository.
+4. Make changes to the codebase.
+6. Push your changes to your forked repository.
+
+## Submitting Changes
+1. Make sure your changes are up-to-date with the latest changes in the main repository.
+2. Make sure your code is well-tested and passes all unit and integration tests.
+3. It would b nice if you could run e2e tests locally with your own app credentials.
+4. Create a pull request to the main repository with your changes.
+
+## Guidelines
+- Please make sure your pull request includes a clear and descriptive title including the issue number in case of an open issue, and a detailed description of the changes you've made.
+- If you're adding new features or functionality, please provide a clear explanation of what the feature does and how to use it.
+- Please make sure your code is well-documented, follows our code style guide, and includes unit tests.
```

Comment:
We don't really have any documentation standards, so I'm not sure that's fair request. In terms of style, shouldn't our CI catch violations of our eslint rules?